src/sources/statsd/mod.rs

```diff
@@ -152,13 +155,13 @@ impl Deserializer for StatsdDeserializer {
             .and_then(parse)
         {
             Ok(metric) => {
-                emit!(&StatsdEventReceived {
-                    byte_size: bytes.len()
+                emit!(&StatsdEventsReceived {
```

Comment:
With #11124 moving `EventsReceived` into places like `build_unix_data_stream` and `TcpSource`, I believe this will cause double counting issues. 

However, if this is removed, statsd `UDP` mode looks like it'll be missing `EventsReceived`; it may also still be missing `BytesReceived`? Sorry if I'm missing it somewhere. It would be helpful for reviewing these instrumentation PRs if you could point out where relevant internal events are emitted since it's not always clear from the changes.

Reply:
I guess this PR need to be rebased on top of what is done in #11124. I'll wait that is get merged to proceed 😉